circle.yml

```diff
@@ -18,3 +18,6 @@ test:
     - nvm use 4; npm rebuild &> /dev/null; npm run testci
     - nvm use 6; npm rebuild &> /dev/null; npm run testci
     - nvm use 8; npm rebuild &> /dev/null; npm run testci
+  post:
+    - npm test
```

Comment:
Do we need to run `npm test` here? Lines 18-20 are already running a test command as well.

Reply:
Yes in fact. But again the same problem i experienced . It works on my local but it seems over circle ci it doesn't generate any report. Ideally any JS inside of package.json should be executed via npm test automatically by circleCi. So to test it out i had to run that cmd individually just to ensure if reports are generating. Now i will make it standard .